project/AppDependencies.scala

```diff
@@ -9,7 +9,9 @@ object AppDependencies {
 
     "uk.gov.hmrc"             %% "govuk-template"           % "5.45.0-play-26",
     "uk.gov.hmrc"             %% "play-ui"                  % "8.4.0-play-26",
-    "uk.gov.hmrc"             %% "bootstrap-play-26"        % "1.3.0"
+    "uk.gov.hmrc"             %% "bootstrap-play-26"        % "1.3.0",
```

Comment:
`bootstrap-play-26` is no longer needed

Reply:
Hi Mac, it is being used in manage-external-guidance view. The BackendController is used by the ScratchController.

Reply:
👍